src/lib/api/configuration-shell-api/configurationShellApiInterface.ts

```diff
@@ -0,0 +1,25 @@
+import { Submodel } from '@aas-core-works/aas-core3.0-typescript/dist/types/types';
+
+export interface ConfigurationShellApiInterface {
+    getIdGenerationSettings(): Promise<Submodel>;
+
+    processGetIdGenerationSettings(response: Response): Promise<Submodel>;
+
+    putSingleIdGenerationSetting(
+        idShort: string,
+        bearerToken: string,
+        values: {
+            prefix: string;
+            dynamicPart: string;
+        },
+    ): Promise<void>;
```

Comment:
Should either return a Response or throw. In either way, will not return void. Can and should be deleted

Reply:
The problem is that the implemented method does two puts, so we can't simply return the response of the request. This is due to how the configuratinShellApi is build. I think refactoring this exceedes the scope of this ticket/story.